react/FilterNavigator.js

```diff
@@ -159,6 +163,14 @@ const FilterNavigator = ({
     (facet) => facet.selected
   )
 
+  const hasFiltersApplied = window.location.href.split('?').length > 1
+
+  const handleResetFilters = () => {
+    const url = window.location.href
+
+    window.location.href = url.split('?')[0]
+  }
+
```

Comment:
Have you ever tried to use the render runtime hook?
https://github.com/vtex-apps/render-runtime/blob/1aa78341198fed752c4785aadce04e38d7ccdae5/react/components/RenderContext.tsx#L13

I don't ever use it, but I think it has these functionalities built-in.

Reply:
Can you provide me an example please? How should I use this?

Reply:
Fixed